src/action.js

```diff
@@ -6,25 +6,50 @@ export async function run() {
   core.info(`
   *** ACTION RUN - START ***
   `)
+  const MAX_MESSAGE_LENGTH = 280
+  const message = core.getInput('message')
+  const appKey = core.getInput('twitter-app-key')
+  const appSecret = core.getInput('twitter-app-secret')
+  const accessToken = core.getInput('twitter-access-token')
+  const accessSecret = core.getInput('twitter-access-token-secret')
+
+  if (!message || !appKey || !appSecret || !accessToken || !accessSecret) {
```

Comment:
I think there's a better way to do this. First of all the inputs can be marked as required when configured in the action xml file. Second, core.getInput has an overload which allow specifying a second argument which, I believe, is a simpler way to do this validation

Reply:
Will look into this.